server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1896,9 +1896,11 @@ public void deleteConsumer(@Verify(Consumer.class) String uuid) {
 
         // we want to insert the content access cert to this list if appropriate
         try {
-            Certificate cert = this.contentAccessManager.getCertificate(consumer);
+            ContentAccessCertificate cert = this.contentAccessManager.getCertificate(consumer);
```

Comment:
This follows with what I was saying above: this bit of code shouldn't change, as it is not interested in details about the SCA cert itself, and is just serializing it as a generic cert.

Reply:
Done.